src/app/pages/ArticlePage/Byline/index.styles.tsx

```diff
@@ -102,6 +103,40 @@ export default {
       },
     }),
 
+  imageLtr: () =>
+    css({
+      display: 'block',
+      width: `${pixelsToRem(80)}rem`,
+      height: `${pixelsToRem(80)}rem`,
+      border: `solid ${pixelsToRem(1)}rem #979797`,
+      margin: `${pixelsToRem(25)}rem ${pixelsToRem(8)}rem ${pixelsToRem(
+        16,
+      )}rem 0px`,
+      backgroundColor: '#d8d8d8',
+      overflow: 'hidden',
+      float: 'left',
+    }),
```

Comment:
Could probably extract out the common CSS between the RTL and LTR styles into an object and then just spread them between the 2 classes.